crates/cheatnet/src/execution/cheatable_syscall_handler.rs

```diff
@@ -93,13 +93,12 @@ impl HintProcessorLogic for CheatableSyscallHandler<'_> {
             }
         }
 
-        match contract_print(vm, maybe_extended_hint) {
-            PrintingResult::Printed => return Ok(()),
-            PrintingResult::Passed => (),
-            PrintingResult::Err(err) => return Err(err),
-        }
-        self.syscall_handler
-            .execute_hint(vm, exec_scopes, hint_data, constants)
+        ContractExecutionSyscallHandler::wrap(self).execute_hint(
+            vm,
+            exec_scopes,
+            hint_data,
+            constants,
+        )
```

Comment:
This shouldn't be created on request, it should be created when `CheatableSyscallHandler` and take it as an constructor arg

Reply:
Done